hack/upload-tutorial-resources.sh

```diff
@@ -20,23 +20,48 @@ then
     exit
 fi
 
+# this array consists of <oci-path of blueprint>;<local path of blueprint>;<default version tag>
```

Comment:
can you add a short section/newdoc in `docs/development` that explain how to add/update the resources and use that file

Reply:
Done.